pandas/conftest.py

```diff
@@ -790,6 +791,17 @@ def tick_classes(request):
     return request.param
 
 
+index_or_series_params = [pd.Index, pd.Series]  # type: IndexOrSeries
+
+
+@pytest.fixture(params=index_or_series_params, ids=["series", "index"])
```

Comment:
Was this inferring as `List[PandasObject]` when used directly as an argument to `params`?

Reply:
Yes, unfortunately.